lib/three_scale_api/tools.rb

```diff
@@ -52,8 +52,8 @@ def self.log_factory(log_level: 'debug')
     #
     # @param [Hash] response Response from server
     def self.check_response(response)
-      if (response.is_a? Hash) && !response['error'].nil?
-        raise APIResponseError.new(response), response['error']
+      if (response.is_a? Hash) && (!response['error'].nil? || !response['errors'].nil?)
```

Comment:
`!response['error'].nil?` is almost the same as `response['error']`. I'd suggest just doing `response['error']` which looks way better.